controllers/propagator/propagation.go

```diff
@@ -814,7 +856,7 @@ func (r *PolicyReconciler) handleDecision(instance *policiesv1.Policy, decision
 // a helper to quickly check if there are any templates in any of the policy templates
 func policyHasTemplates(instance *policiesv1.Policy) bool {
 	for _, policyT := range instance.Spec.PolicyTemplates {
-		if templates.HasTemplate(policyT.ObjectDefinition.Raw, startDelim) {
+		if templates.HasTemplate(policyT.ObjectDefinition.Raw, startDelim, encryptionEnabled) {
```

Comment:
You always want the last parameter to `false` because setting it to `true` checks for already encrypted strings, which is used on the Config Policy Controller. It wouldn't apply here since the Propagator will not be performing decryption.